CMakeLists.txt

```diff
@@ -233,7 +233,7 @@ add_glob_target(
   TARGET lintlua-stylua
   COMMAND ${STYLUA_PRG}
   FLAGS --color=always --check --respect-ignores
-  GLOB_DIRS runtime/
+  GLOB_DIRS runtime/ scripts/
```

Comment:
I think both this and the `GLOB_DIRS` in `formatlua` needs a "src". Otherwise there will be a discrepancy between `make formatlua` (which won't format `src`) and a top-level `stylua` (which will format `src`). Basically, for every entry we remove from `.styluaignore` we should add it to the stylua targets.

Reply:
Thanks for the note. src/ has some churn, will do it in a follow PR. Reverted the .styleluaignore change in this PR.